### ISSUE TITLE
pycham Cannot auto replenish kersa.layers

### ISSUE THREAD

**Issue by a906247390** [tensorflow/tensorflow#52755]
import numpy as np
import pandas as pd
import matplotlib as plt
import tensorflow as tf
import minst_reader

# 加载fashion_mnist数据集
x_train, y_train= minst_reader.load_mnist('fashion',kind='train')
x_test, y_test= minst_reader.load_mnist('fashion',kind='t10k')
x_train,y_train = x_train/ 255.0,y_train/255.0

model = tf.keras.Sequential([
    tf.keras.layers.Flatten(),
    tf.keras.layers.Dense(128,activation = 'relu')
    tf.keras.layers.Dense(10,activation = 'softmax')
])
![}8C4{Z5%P`2%A)JP3OB{7AN](https://user-images.githubusercontent.com/72368814/139173294-0c713a9e-9849-4bbc-a3c3-a12fe62009cf.png)
e
